pages/support/index.html

```diff
@@ -1,33 +1,25 @@
 ---
 layout: default
-title: Gruntwork | Support
-excerpt: Support.
+title: Gruntwork Support
+excerpt: Expert support, automatic updates, and security alerts for the Infrastructure as Code Library and Reference Architecture.
 permalink: /support/
-slug: support
-footer_heading: Ready to hand off the Gruntwork?
+slug: subscriptions
+footer_heading: Pick a support plan now!
```

Comment:
This footer should probably redirect to the pricing page since you can't select a support plan on the contact us page. Alternatively, reword this to something that makes sense for contact us.

Reply:
Good catch. Fixed: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/23/commits/c623e1b77209af049b96c10ec6e351fe635c8063